### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+The `did:tdw` (Trust DID Web) method is an enhancement to the
+`did:web` protocol, providing a complementary web-based DID method that addresses limitations
+of `did:web`. It's features include the following.
+
+- Ongoing publishing of all DID Document (DIDDoc) versions for a DID instead of,
+  or alongside a `did:web` DID/DIDDoc.
+- Uses the same DID-to-HTTPS transformation as `did:web`.
+- Provides resolvers the full history of the DID using a verifiable chain of
+  updates to the DIDDoc from genesis to deactivation.
+- A [[def: self-certifying identifier]] (SCID) for the DID that is globally
```
Comment (brianorwhatever) [inline README.md:23]:
```suggestion
- A self-certifying identifier (SCID) for the DID that is globally
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+  the DID's history.
+- DIDDoc updates include a proof signed by the DID Controller(s) *authorized* to
+  update the DID.
+- An optional mechanism for publishing "pre-rotation" keys to prevent loss of
+  control of the DID in cases where an active private key is compromised.
+- DID URL path handling that defaults (but can be overriden) to automatically
+  resolving `<did>/path/to/file` by using a comparable DID-to-HTTPS translation
+  as for the DIDDoc.
+- A DID URL path `<did>/whois` that defaults to automatically returning (if
+  published by the DID controller) a [[ref: Verifiable Presentation]] containing
```
Comment (brianorwhatever) [inline README.md:35]:
```suggestion
  published by the DID controller) a Verifiable Presentation containing
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+  [[ref: Verifiable Credentials]] with the DID as the `credentialSubject`,
```
Comment (brianorwhatever) [inline README.md:36]:
```suggestion
  Verifiable Credentials with the DID as the `credentialSubject`,
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,69 @@
...
+  signed by the DID.
+
+Combined, the additional feature enable greater trust and security without
```
Comment (brianorwhatever) [inline README.md:39]:
```suggestion
Combined, the additional features enable greater trust and security without
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,69 @@
...
+compromising the simplicity of `did:web`. The incorporation of the DID Core
+compatible "/whois" path, drawing inspiration from the traditional WHOIS
+protocol [[spec:rfc3912]], offers an easy to use, decentralized, trust registry.
```
Comment (brianorwhatever) [inline README.md:42]:
```suggestion
protocol, offers an easy to use, decentralized, trust registry.
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,69 @@
...
+This `did:tdw` aims to establish a more trusted and secure web environment by
```
Comment (brianorwhatever) [inline README.md:43]:
```suggestion
The `did:tdw` method aims to establish a more trusted and secure web environment by
```

**Review by brianorwhatever — DISMISSED** [review]

**Comment by swcurran** [comment]
Doh…thanks!  Fixed.

**Comment by swcurran** [comment]
Missed a few of your updates, so need another review, please.

**Comment by brianorwhatever** [comment]
We will just merge these in through #41

**Pull request closed without merging**